include/openssl/configuration.h.in

```diff
@@ -72,4 +72,7 @@ extern "C" {
 }
 # endif
 
+/* Features completely removed */
+#define OPENSSL_NO_EC2M
```

Comment:
Makes sense here. Should it be wrapped into a `#ifndef` guard, so people who set it don't redefine it?

Reply:
I'm not quite sure what difference would happen there, but you probably have more experience with that kind of mess than I do. Will add.